app-router/localization.mdx

```diff
@@ -0,0 +1,87 @@
+---
+title: Localization
+description: "This guide walks you through setting up localization for your Makeswift site so you can visually customize your site for different locales."
+---
+
+import AppCatchAllLocalizedExample from "/snippets/app-router/catch-all-localized.mdx"
+import LocalizationExample from "/snippets/app-router/localization.mdx"
+
+<Info>
+  The feature is deeply integrated with [Next.js' internationalization
+  features](https://nextjs.org/docs/app/building-your-application/routing/internationalization).
+</Info>
+
+<Warning>
+  Localization is currently only available on the Enterprise plan. Reach out to
+  [support@makeswift.com](mailto:support@makeswift.com) if you want
+  to enable this feature.
+</Warning>
+
+## Getting started
+
+<Steps>
+
+<Step title="Add locales in the Makeswift builder">
+Open site settings and go to the "Locales" tab:
+
+![Localization first step](../images/localization-first-step.png)
+
+To add a new locale, click the "+ Add locale" button. You can modify or delete existing locales by hovering over the locale:
+
+![Hovering over the locale](../images/localization-hover-locale.png)
+
+You can also modify the default locale by hovering over the default locale and clicking the edit button.
+
+Once you add all the locales you need, it might look like this:
+
+![Manage locales in settings](../images/localization-manage-locales.png)
+
+</Step>
+
+<Step title="Configure your Next.js app for localization">
+
+Follow the [internationalization docs](https://nextjs.org/docs/app/building-your-application/routing/internationalization) for app router and make sure that all routes except the [Makeswift api handler](/reference/makeswift-api-handler) are nested inside `/app/[[lang]]`.
```

Comment:
I thought we were doing `/app/[lang]` instead of `/app/[[lang]]`?

Reply:
Ok, so we seem to be using `/app/[[locale]]` in the actual file snippet. Seems like we should make it a non-optional catch all route and then change the docs to reference `locale` as the slug instead of `lang`?

Reply:
In fact, given the snippet does `lang: string` in the TypeScript types, the file path should be `/app/[lang]`. It is currently `/app/[[locale]]` which is wrong in terms of the code. If we tried to follow the guide we'd run into problems. We should try following the guide to make sure there's no errors like these.

Reply:
👍